ProboCI/probo-image-builder
# Feature/lliss/php versions

This change restructures the ansible roles to break them into smaller chunks. It further adds builds for php7.0 and 7.1. The result is that you will now find nightlies for three different docker images after running `vagrant up` the first time or `vagrant provision` on subsequent runs.

## To test
- See https://github.com/ProboCI/probo-image-builder/pull/6 as the instructions are similar with respect to updating probo to accept the new images.
